hack/copyright.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
I used bash because I find the error handling much better and way more reliable, set -eu doesn't work equally in bash and sh -o pipefail is not really supported in dash. Also, and this is an excuse, I like having the `if [[  ]]`.

The CI already uses bash and plenty of CNCF projects, like kubernetes require bash... I'm not quite sure on this day and age the POSIX shell is really much more portable to be honest. If we REALLY care about portability I'll change it but quite frankly most people has bash nowadays and I certainly see some benefits in error handling.